common/css.c

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright © 2017-2018 The Crust Firmware Authors.
+ * SPDX-License-Identifier: (BSD-3-Clause OR GPL-2.0)
+ */
+
+#include <compiler.h>
+#include <css.h>
+#include <debug.h>
+#include <error.h>
+#include <stdint.h>
+#include <util.h>
+
+/*
+ * These generic functions provide sane default implementations of the CSS API,
+ * so platforms may incrementally implement power domain control functions.
+ */
+uint8_t __weak
+css_get_css_state(void)
```

Comment:
why do we need this function if we can always assume power state is on?

Reply:
Because once we implement turning it off (with a platform-specific implementation of this function), we can't assume that anymore